debian/changelog

```diff
@@ -1,3 +1,10 @@
+opx-nas-interface (5.18.0+opx6) unstable; urgency=medium
+
+  * Update: VRF get-handler support
+  * Update: VRF configuration script
+
```

Comment:
plz mention about t he bugfix for vrf creation failure

Reply:
will do